packages/stryker/src/reporters/ProgressKeeper.ts

```diff
@@ -24,5 +26,23 @@ abstract class ProgressKeeper implements Reporter {
       this.progress.survived++;
     }
   }
+
+  protected getEtc() {
+    const timeLeft = Math.floor(this.timer.elapsedSeconds() / this.progress.tested * (this.progress.total - this.progress.tested));
+
+    if (isFinite(timeLeft)) {
+      const hours = Math.floor(timeLeft / 3600);
+      const minutes = Math.floor((timeLeft / 60) - (hours * 60));
+      const seconds = Math.floor(timeLeft - (hours * 3600) - (minutes * 60));
+
+      let output = (hours > 0) ? `${hours}h, ` : '';
+      output += (minutes > 0) ? `${minutes}m, ` : '';
```

Comment:
When hours is greater than 0, but minutes is 0 it means that minutes is left out. Though technically correct, i think it would be clearer if minutes is displayed. For example: `1h, 0m, 20s` instead of `1h, 20s`.

Suggestion: 

```js
if (hours > 0) {
  return `${hours}h, ${minutes}m, ${seconds}s`;
} else if(minutes > 0) {
  return `${minutes}m, ${seconds}s`;
} else {
  return `${seconds}s`
}
```